### DISCUSSION TITLE
Wrong release shown on release page

### DISCUSSION THREAD

**Discussion by bdlink** [okd-project/okd#1324] [Releases]
Documentation issue.

There seem to be many pages showing our releases.  I have been giving students the following URL:
https://github.com/openshift/okd/releases which now goes to https://github.com/okd-project/okd/releases (Good).

Problem is the page shows 4.10.0-0.okd-2022-05-28-062148 as the release and you have to scroll down to find the current 4.11.0-0.okd-2022-08-20-022919.  The current release should be at the top.

I have been using the release status page at https://origin-release.apps.ci.l2s4.p1.openshiftapps.com/ to check on a daily basis.   Is this still the best page to use for stable releases?  

The "client tools" page looks more useful for the general public, so the top release should be our current release.

**Comment by vrutkovs** [comment]
>Problem is the page shows 4.10.0-0.okd-2022-05-28-062148 as the release and you have to scroll down to find the current 4.11.0-0.okd-2022-08-20-022919. The current release should be at the top.

This is some Github quirk - turns out we had a copy of 4.10.0-0.okd-2022-05-28-062148 in drafts - and GitHub was showing it first. I deleted the draft (4.10.0-0.okd-2022-05-28-062148 is still there) and now latest 4.11 shows up as latest

**Reply by bdlink** [reply]
Thanks Vadim, and my 24 students will be grateful for less confusion in their lives.
